packages/account/src/Sections/Security/ClosingAccount/closing-account-reason-form.tsx

```diff
@@ -0,0 +1,266 @@
+                                    'If you don’t mind sharing, which other trading platforms do you use?'
+                                )}
+                                name='other_trading_platforms'
+                                value={values.other_trading_platforms}
+                                max_characters={CHARACTER_LIMIT}
+                                onChange={e => handleInputChange(e, values.other_trading_platforms, handleChange)}
+                                onPaste={handleInputPaste}
+                            />
+                        )}
+                    </Field>
+                    <Field name='do_to_improve'>
+                        {({ field }: FieldProps) => (
+                            <Input
+                                {...field}
+                                className='closing-account-reasons__input'
+                                data-lpignore='true'
+                                autoComplete='off'
+                                type='textarea'
+                                placeholder={localize('What could we do to improve?')}
+                                name='do_to_improve'
+                                value={values.do_to_improve}
+                                max_characters={CHARACTER_LIMIT}
+                                onChange={e => handleInputChange(e, values.do_to_improve, handleChange)}
+                                onPaste={handleInputPaste}
+                            />
+                        )}
+                    </Field>
+                    <div className='closing-account-reasons__footer'>
+                        <div className='closing-account-reasons__hint-wrapper'>
+                            <Text size='xxs' as='p' color='less-prominent' className='closing-account-reasons__hint'>
+                                <Localize
+                                    i18n_default_text='Remaining characters: {{remaining_characters}}'
+                                    values={{ remaining_characters }}
+                                />
+                            </Text>
+                            {Object.keys(errors).length > 0 &&
+                                Object.entries(errors).map(([key, value]) => (
+                                    <Text
+                                        as='p'
+                                        weight='bold'
+                                        size='xs'
+                                        color='loss-danger'
+                                        className='closing-account-reasons__error'
+                                        key={key}
+                                    >
+                                        {value}
+                                    </Text>
+                                ))}
+                        </div>
+                        <FormSubmitButton
+                            is_disabled={!dirty || Object.keys(errors).length > 0}
```

Comment:
```suggestion
                            is_disabled={!dirty || !isValid}
```

`isValid` does the same thing as checking if `errors` has length or 

https://formik.org/docs/api/formik#isvalid-boolean